app/helpers/companies_helper.rb

```diff
@@ -5,9 +5,7 @@ def payment_visible_for_user?(user, company)
   end
 
   def list_categories company, separator=' '
-    if company.business_categories.any?
-      company.business_categories.includes(:shf_applications).map(&:name).sort.join(separator)
-    end
+    company.business_categories.order(:name).pluck(:name).join(separator)
```

Comment:
there is method categories_names on company

Reply:
I am now using that in my PR.  Thanks for the reminder.